wd_core.au3

```diff
@@ -843,30 +843,32 @@ EndFunc   ;==>_WD_ElementAction
 ; #FUNCTION# ====================================================================================================================
 ; Name ..........: _WD_ExecuteScript
 ; Description ...: Execute Javascipt commands.
-; Syntax ........: _WD_ExecuteScript($sSession, $sScript[, $sArguments = Default[, $bAsync = Default]])
-; Parameters ....: $sSession   - Session ID from _WD_CreateSession
-;                  $sScript    - Javascript command(s) to run
-;                  $sArguments - [optional] String of arguments in JSON format
+; Syntax ........: _WD_ExecuteScript($sSession, $sScript[, $sArguments = Default[, $bAsync = Default[, $bReturnAsValue = False]]])
+; Parameters ....: $sSession            - Session ID from _WD_CreateSession

```

Comment:
I think these extra spaces will cause issues with building the help file

Reply:
looking into:
https://github.com/Danp2/WebDriver/pull/110/files
all is fine